src/infrasys/time_series_metadata_store.py

```diff
@@ -387,8 +460,8 @@ def _make_where_clause(
             ts_str = "AND time_series_type = ?"
             params.append(time_series_type)
 
-        if attribute_hash is None and user_attributes:
-            ua_hash_filter = _make_user_attribute_filter(user_attributes, params)
+        if attribute_hash is None and features:
```

Comment:
Can't we completely remove attribute_hash?

Reply:
yes. I will remove it.